src/main/java/org/mskcc/cbio/oncokb/service/SlackService.java

```diff
@@ -553,36 +521,23 @@ private List<LayoutBlock> buildUserInfoBlocks(UserDTO user) {
     private List<LayoutBlock> buildAdditionalInfoBlocks(UserDTO userDTO, boolean trialAccountActivated, ActionId actionId) {
         List<LayoutBlock> layoutBlocks = new ArrayList<>();
 
-        if (withForProfitClarificationNote(userDTO, actionId)) {
-            layoutBlocks.add(buildPlainTextBlock("We have sent a clarification email to the user asking why they are applying for the academic license while affiliated with a for-profit company.", FOR_PROFIT_CLARIFICATION_NOTE));
-        }
-        if (withAcademicClarificationNote(userDTO, actionId)) {
-            layoutBlocks.add(buildPlainTextBlock("We have sent a clarification email to the user asking why they could not use an institution email to register.", ACADEMIC_CLARIFICATION_NOTE));
-        }
-        if (withUseCaseClarificationNote(userDTO, actionId)) {
-            layoutBlocks.add(buildPlainTextBlock("We have sent a clarification email to the user asking to further explain their use case.", USE_CASE_CLARIFICATION_NOTE));
-        }
-        if (withDuplicateUserClarificationNote(userDTO, actionId)) {
-            layoutBlocks.add(buildPlainTextBlock("We have sent a clarification email to the user asking why they registered multiple accounts.", DUPLICATE_USER_CLARIFICATION_NOTE));
-        }
-        if (withRegistrationInfoClarificationNote(userDTO, actionId)) {
-            layoutBlocks.add(buildPlainTextBlock("We have sent a clarification email to the user asking to provide detailed registration info.", REGISTRATION_INFO_CLARIFICATION_NOTE));
-        }
-        if (withLicenseOptionsNote(userDTO, actionId)) {
-            layoutBlocks.add(buildPlainTextBlock("We have sent an email to the user with license options for their affiliated company.", LICENSE_OPTIONS_NOTE));
+        for (MailType inquiryMail : Arrays.stream(MailType.values()).filter(mt -> mt.getCategory().orElse(null) == EmailCategory.CLARIFY || mt.getCategory().orElse(null) == EmailCategory.LICENSE).collect(Collectors.toList())) {
```

Comment:
this is how we iterate through enum, can filter by clarification/license emails like here.